misaligned.cpp

```diff
@@ -16,6 +21,7 @@ int printColorMap() {
 int main() {
     int result = printColorMap();
     assert(result == 25);
+    assert(ComputedColorCode(0,0) == 1);

```

Comment:
Good isolation of the computation.
There is still an error in the formatting, since the | symbols are not aligned as a table. Can the formatting be isolated in a similar way to make it testable?
Hint: Use a function to format and return a row as a string. Then do the court separately.